project/MimaVersionPlugin.scala

```diff
@@ -77,6 +82,18 @@ object MimaVersionPlugin extends AutoPlugin {
 
   override def projectSettings: Seq[Setting[_]] = Seq(
     isMimaEnabled := false,
+    mimaBinaryIssueFilters ++= Seq(
```

Comment:
A few minimal MiMa filters. Some are there because MiMa seemed to be confused by some of the code, the `default` one is there because I made the mistake of using default parameters somewhere I shouldn't have. This does imply that there's one method (in a non internals package) that's actually not binary compatible in there, but really, it should not be considered API method by anyone.